octopoes/octopoes/models/ooi/findings.py

```diff
@@ -112,6 +150,13 @@ def format_reference_human_readable(cls, reference: Reference) -> str:
 
 
 class MutedFinding(OOI):
+    """Represents muted findings.
```

Comment:
Not sure if "Represents muted findings." is insightful here. Could maybe explain what the object does, for example:
 
MutedFindings can be attached to findings, this will make the findings not show up in the Findings table" - or something similar.